src/styles/blocks/content.scss

```
@@ -0,0 +1,11 @@
+.content {
+  display: grid;
+  max-width: 1200px;
+  grid-template-columns: repeat(4, 200px);
+  margin: 0 auto;
+  justify-content: center;
+  column-gap: 48px;
+  &__cards {
+    margin-top: 50px;
+  }
+}
```

no need for auto margin, you center your cards with justify-content property
also, now when you use grid use row-gap for vertical spacing , not margins

```suggestion
.content {
  display: grid;
  max-width: 1200px;
  grid-template-columns: repeat(4, 200px);
  justify-content: center;
  column-gap: 48px;
  row-gap: 50px;
}
```